config.go

```diff
@@ -137,6 +137,9 @@ type Config struct {
 	FSMonitoring bool `toml:"fs_monitoring" comment:"Turn on/off any disk- and filesystem-related monitoring like fill levels and iops"`
 
 	NetMonitoring bool `toml:"net_monitoring" comment:"Turn on/off any network-related monitoring"`
+
+	OnHTTP5xxRetries       int     `toml:"on_http_5xx_retries" comment:"Number of retries if server replies with a 5xx code"`
+	OnHTTP5xxRetryInterval float64 `toml:"on_http_5xx_retry_interval" comment:"Interval between retries to contact server in case of a 5xx code"`
```

Comment:
I think having a measurement name in field comment would be great. It must be 'seconds' because all other fields uses it.

Reply:
Fixed